src/test/java/core/basesyntax/strategy/handlers/SupplyOperationHandlerTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.strategy.handlers;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.db.Storage;
+import org.junit.After;
+import org.junit.Test;
+
+public class SupplyOperationHandlerTest {
+    private SupplyOperationHandler handler = new SupplyOperationHandler();
+
+    @After
+    public void afterEachTest() {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void apply_AddNewFruit_Ok() {
```

Comment:
```suggestion
    public void apply_addNewFruit_ok() {
```
likewise in the following cases